package.json

```diff
@@ -9,7 +9,7 @@
     "licenses": "yarn licenses generate-disclaimer > 3RD_PARTY_LICENSES",
     "markdownlint": "markdownlint *.md ./github/**/*.md",
     "shellcheck": "shellcheck .github/workflows/*.sh",
-    "test": "node test/setup-beam.test.js",
+    "test": "node --test",
```

Comment:
I think this'll go away as soon as we move the new `lib/` file back into `setup-beam.js`...

Reply:
Check https://github.com/erlef/setup-beam/pull/328#issuecomment-2832603425.